qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -453,4 +454,52 @@ def test_snap_schedule_with_mgr_restart(self):
         # cleanup
         self.fs_snap_schedule_cmd('remove', path=testdir, snap_schedule='1M')
         self.remove_snapshots(testdir[1:])
-        self.mount_a.run_shell(['rmdir', testdir[1:]])
+        self.mount_a.run_shell(['rmdir', testdir[1:]])    
+
+class TestSnapSchedulesSnapdir(TestSnapSchedulesHelper):
+    def remove_snapshots(self, dir_path, sdn):
+        snap_path = f'{dir_path}/{sdn}'
+
+        snapshots = self.mount_a.ls(path=snap_path)
+        for snapshot in snapshots:
+            snapshot_path = os.path.join(snap_path, snapshot)
+            log.debug(f'removing snapshot: {snapshot_path}')
+            self.mount_a.run_shell(['rmdir', snapshot_path])
+
+    def get_snap_dir_name(self):
+        from tasks.cephfs.fuse_mount import FuseMount
+        from tasks.cephfs.kernel_mount import KernelMount
+
+        sdn = self.fs.get_config('client_snapdir')
```

Comment:
From the test, the snapdir should be `.customsnapfuse`:

```
  overrides:
    admin_socket:
      branch: wip-vshankar-testing1-20220603-134300
    ceph:
      conf:
        client:
          client mount timeout: 600
          client snapdir: .customsnapfuse
```
But it was trying to find the `.snap` instead:

```
2022-06-03T11:45:54.776 DEBUG:teuthology.orchestra.run.smithi175:> (cd /home/ubuntu/cephtest/mnt.0 && exec ls snap_test_dir1/.snap)
2022-06-03T11:45:54.838 INFO:teuthology.orchestra.run.smithi175.stderr:ls: cannot access 'snap_test_dir1/.snap': No such file or directory
2022-06-03T11:45:54.839 DEBUG:teuthology.orchestra.run:got remote process result: 2
```

It's buggy here. It will use the default `.snap` value always. Please see https://github.com/ceph/ceph/blob/main/src/common/options/mds-client.yaml.in#L95-L103